Description: Patch to remove unnecessary link to libdl
Debian Bug: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=540599
Author: Fabio Tranchitella <kobold@debian.org>

Index: 0.8/auto/os/features
===================================================================
--- 0.8.orig/auto/os/features	2010-10-31 07:32:20.408338001 +0000
+++ 0.8/auto/os/features	2010-10-31 07:32:26.528338002 +0000
@@ -252,28 +252,6 @@
 . auto/feature
 
 
-ngx_feature="dlopen()"
-ngx_feature_name=
-ngx_feature_run=no
-ngx_feature_incs="#include <dlfcn.h>"
-ngx_feature_path=
-ngx_feature_libs=
-ngx_feature_test="dlopen(NULL, 0)"
-. auto/feature
-
-
-if [ $ngx_found != yes ]; then
-
-    ngx_feature="dlopen() in libdl"
-    ngx_feature_libs="-ldl"
-    . auto/feature
-
-    if [ $ngx_found = yes ]; then
-        NGX_LIBDL="-ldl"
-    fi
-fi
-
-
 ngx_feature="sched_yield()"
 ngx_feature_name="NGX_HAVE_SCHED_YIELD"
 ngx_feature_run=no
